Jamulus.pro

```diff
@@ -125,19 +125,19 @@ win32 {
             message(Using ASIO.)
             message(Please review the ASIO SDK licence.)
 
-            !exists(windows/ASIOSDK2) {
-                error("Error: ASIOSDK2 must be placed in Jamulus windows folder.")
+            !exists(libs/ASIOSDK2) {
```

Comment:
I think it is better to check for libs/ASIOSDK2/common 

That's conform compiling.md "so that, e.g., the folder `[\path\to\jamulus\source]\libs\ASIOSDK2\common` exists" )

Reply:
Fixed.